src/api-reference/request/v4.endpoints.markdown

```diff
@@ -660,42 +715,21 @@ id|string|-|**Required** Unique identifier of the Expense type
 name|string|-|Name of the Expense type
 href|string|[RFC 3986](https://tools.ietf.org/html/rfc3986)|Hyperlink to the resource for the Expense type definition
 
-### <a name="schema-requesttype"></a>Request Type
-
-Name|Type|Format|Description
----|---|---|---
-code|string|-|The code of the type inherited from the Request Policy type
-label|string|-|The label of the type inherited from the Request Policy Type
-
-### <a name="schema-travelallowance"></a>Travel Allowance
-
-Name|Type|Format|Description
----|---|---|---
-dailyTravelAllowanceId|string|-|The fixed daily travel allowance id associated with the Expense
-
-### <a name="schema-tripdata"></a>Trip Data
+### <a name="schema-listitemfield"></a>List Item Field
 
 Name|Type|Format|Description
 ---|---|---|---
-agencyBooked|boolean|-|True if this travel is (or has to be) handled by a travel agency
-legs|list|[Segment Leg](#schema-segmentleg)|The list of the legs of the travel
-tripType|string|-|Indicates the type of this trip. Should be one of `ONE_WAY`, `ROUND_TRIP` or `MULTI_STOPS`. If not provided, will be detected from the given legs.
-segmentType|object|[Segment Type](#schema-segmenttype)|**Required** The type of the segment
-selfBooked|boolean|-|True if this travel has been reserved by Concur Travel, or if Concur Travel has retrieved the trip information in the GDS
-
-### <a name="schema-vendor"></a>Vendor
-
-Name|Type|Format|Description
----|---|---|---
-id|string|-|The vendor identifier of the entry
-name|string|-|The vendor description of the entry
+code|string|-|The short code of the list item
+value|string|-|Name of the Expense type
+href|string|[RFC 3986](https://tools.ietf.org/html/rfc3986)|Hyperlink to the resource for the list item
 
 ### <a name="schema-segmentleg"></a>Segment Leg
 
 Name|Type|Format|Description
 ---|---|---|---
-comment|string|-|Contains the last comment saved in this segment leg.
+class|list|[List Item Field](#schema-listitemfield)|The booking class of the segment leg, class part of the CLASS_LI_KEY table.
```

Comment:
@laurentlhermine 
"The booking class of the segment leg, class part of the CLASS_LI_KEY table" could be eventually replaced by "The booking class of the segment leg." without necessarily adding the technical information of the table reference. 
This might brought some question if the client uses the field "class" in the segment leg linked to a custom table (and not the CLASS_LI_KEY table) as it will not work, but even in the audit rule technical guide there is the class field and class of service field respectively referring to the class of the segment leg and class of service of the segment leg. Let's delete the CLASS_LI_KEY reference in the comment and we'll see if this raises some use cases.